src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -111,7 +111,80 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
     // Receive message from the webview.
     webviewPanel.webview.onDidReceiveMessage(e => {
       switch (e.type) {
-        // TODO Add more webview modification handlers
+        case 'setParam':
+          if (this._oneConfig[e.section] === undefined) {
+            this._oneConfig[e.section] = {};
+          }
+          if (this._oneConfig[e.section][e.param] === undefined) {
+            this._oneConfig[e.section][e.param] = '';
+          }
+          this._oneConfig[e.section][e.param] = e.value;
+          break;
+        case 'setSection':
+          this._oneConfig[e.section] = ini.parse(e.param);
```

Comment:
if `ini.parse()` is failed?

Reply:
As any documentation is found, I looked into the implementation of `ini.parse`.
But I cannot find any error/exception handling in the function.
So I think `ini.parse` returns wrongly parsed results when `e.param` is invalid..!